lib/logstash/agent.rb

```diff
@@ -321,10 +321,20 @@ def local_config(path)
         encoding_issue_files << file
       end
       config << cfg + "\n"
+      if config_test?
+        @logger.debug? && @logger.debug("\n***** Begin configuration file: #{file}")
```

Comment:
The newlines make it more readable in debug log output, IMO.

I can definitely add the `#`, that's a great idea.  Allows cut/paste testing (but that was the other reason for the newlines).